pygmt/src/grd2cpt.py

```diff
@@ -0,0 +1,180 @@
+"""
+grd2cpt - Create a CPT from a grid file.
+"""
+
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import (
+    build_arg_string,
+    data_kind,
+    dummy_context,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    A="transparency",
+    C="cmap",
+    D="background",
+    F="color_model",
```

Comment:
Missing `-E` (see https://docs.generic-mapping-tools.org/6.1/grd2cpt.html#e). Shall we call the alias `nlevels` like GMT.jl at https://www.generic-mapping-tools.org/GMT.jl/latest/#GMT.grd2cpt?

Reply:
Wrapped `-E` in 3e4b017